packages/Popover/stories/examples/ScrollContainer.js

```diff
@@ -62,15 +65,52 @@ PopoverBox.propTypes = {
   getScrollContainer: PropTypes.func.isRequired,
 };
 
-const ExampleStory = props => (
-  <React.Fragment>
-    <CenteredStory>
-      <ScrollBox id="scroll-container">
-        <PopoverBox {...props} getScrollContainer={() => document.querySelector("#scroll-container")} />
-      </ScrollBox>
-    </CenteredStory>
-    <TrailerBox>Fin.</TrailerBox>
-  </React.Fragment>
-);
+function setKnobsTab() {
+  setTimeout(() => {
+    // eslint-disable-next-line no-restricted-globals
+    parent.document.querySelector("#storybook-panel-root .simplebar-content button").click();
```

Comment:
This selector can break in future versions of storybook, should we let this break then?

Reply:
This is pretty experimental. If it's a pattern we _actually_ want to adopt, we can put it in one central place making it easier to repair. Maybe someday there will even be a supported API for this kind of thing. 

But for now, I will write it more defensively so it can fail more gracefully.  Still up for debate as to whether "we" want to use something like this or not though.